integration-commons/src/test/java/org/symphonyoss/integration/exception/RemoteApiExceptionTest.java

```diff
@@ -0,0 +1,65 @@
+package org.symphonyoss.integration.exception;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link RemoteApiException}.
+ * Created by crepache on 23/06/17.
+ */
+public class RemoteApiExceptionTest {
+
+  private int code = 0;
+
+  private String message = "message";
+
+  private Exception e = new Exception();
+
+  @Test
+  public void testRemoteApiExceptionWithCodeAndMessage() {
+    RemoteApiException exception = new RemoteApiException(code, message);
+    int expectedCode = exception.getCode();
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.